sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server/README.md

```diff
@@ -3,109 +3,102 @@
-   - For **Who can consent**, ensure that the **Admins and users** option is selected.
-   - In the **Admin consent display name** box, enter **Access File.read as a user**.
-   - In the **Admin consent description** box, enter **Accesses the File.read web API as a user**.
-   - In the **User consent display name** box, enter **Access File.read as a user**.
-   - In the **User consent description** box, enter **Accesses the File.read web API as a user**.
-   - For **State**, keep **Enabled**.
-   - Select **Add scope**.
-   ![Scope Config](docs/image-scope-configurations.png "Scope Config")
+### Configure web app
+If you don't know web app at the moment, you can look at this [azure-spring-boot-sample-active-directory-webapp].
+At this point, I believe you have understood the web app, so let's continue!
+1. Add custom apis in the `webapi` to `webapp`.
+2. Under **webapp** application, select **API permissions** -> **Add a permission** -> **My APIS**, select **webapi** tab, add **File.read** permission,click **Add permissions** button.![Add custom apis to webapp](docs/image-add-custom-apis-to-webapp.png "Add custom apis to webapp")
+3. click **Grant admin consent for...**.![Grant permission](docs/image-granted-permission.png "Grant permission")
+4. Manually remove the admin consent for **user_impersonation**.(Ensure that the authorization in `webapp` is not changed.)![Final configuration](docs/image-final.png "Final configuration")
 
-If you still don't understand, you can look at this [register app or web api][Register app or web API] and another [expose scoped permission to web api][Expose scoped permission to web API]. I believe it will also help you.
+See [Configure a client application to access a web api] for more information about web app.
 
 ## Examples
-
-### Configure application.properties
-
-```properties
-azure.activedirectory.client-id=xxxxxx-your-client-id-xxxxxx
-azure.activedirectory.app-id-uri=xxxxxxxx-app-id-uri-xxxxxxxxxx
-azure.activedirectory.session-stateless=true
+### Configure application.yml
+
+Add the following to appapplication.yml in webapp.
+```yaml
+azure:
+  activedirectory:
+    authorization:
+      webapi:
+        scopes: api://xxxxx-xxxxxxx-xxxxxxx/File.read
+  ```
+
+Modify application.yml in webapi.
+```yaml
+#If we configure the azure.activedirectory.client-id or azure.activedirectory.app-id-uri will be to check the audience.
+#In v2.0 tokens, this is always client id of the app, while in v1.0 tokens it can be the client id or the application id url used in the request.
+#If you are using v1.0 tokens, configure both to properly complete the audience validation.
+
+azure:
+  activedirectory:
+    client-id: <client-id>
+    app-id-uri: <app-id-uri>
+```
+**NOTE**：Set the `server.port` in the application.yml of `webapp` and `webapi` to different value.
+
+### Add code
+Add code(Here code is just an example, let you understand the process.) to webapp's `ClientController.java`.
```

Comment:
"Here code is just an example, let you understand the process."
-->
"The code here is just an example to show the process"